packages/firestore/src/local/indexeddb_mutation_queue.ts

```diff
@@ -598,6 +578,58 @@ export class IndexedDbMutationQueue implements MutationQueue {
   }
 }
 
+/**
+ * Delete a mutation batch and the associated document mutations.
+ * @return done, A PersistencePromise that signals when the deletion has completed, and removedDocuments, the document
```

Comment:
Nit: can you reflow this to either 80 or 100 characters? Pick one of your choice. I think we are pretty inconsistent already.

Reply:
Rewrote the comment. But, we should configure prettier to do whatever is the appropriate thing here, it's a silly thing for humans to do.